docs/development/go/go-structures/index.md

```diff
@@ -2,13 +2,14 @@
 author:
   name: Mihalis Tsoukalos
```

Comment:
Hey @mactsouk - I'd like to walk through the changes I made in the course of the tech edit, which will be listed below